src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -121,7 +121,7 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
     }
 
     public async attachNewAccount(): Promise<void> {
-        const defaultExperiencePick = await vscode.window.showQuickPick(getExperienceQuickPicks(), { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
```

Comment:
We don't support attach for postgres? Is there an issue for that?

Reply:
Not for this release, we haven't created support for it yet.

Reply:
https://github.com/microsoft/vscode-cosmosdb/issues/1531